go/tasks/plugins/array/k8s/task.go

```diff
@@ -75,8 +77,26 @@ func (t Task) Launch(ctx context.Context, tCtx core.TaskExecutionContext, kubeCl
 	})
 
 	pod.Spec.Containers[0].Env = append(pod.Spec.Containers[0].Env, arrayJobEnvVars...)
+	var inputReader io.InputReader
+
+	taskTemplate, err := tCtx.TaskReader().Read(ctx)
+	if err != nil {
+		return LaunchError, errors2.Wrapf(ErrGetTaskTypeVersion, err, "Unable to read task template")
+	} else if taskTemplate == nil {
+		return LaunchError, errors2.Wrapf(ErrGetTaskTypeVersion, err, "Missing task template")
+	}
+
+	if taskTemplate.Version == 1 {
```

Comment:
>= ?

Reply:
sorry not following

Reply:
:-D Sorry meant to say ">=" but that got converted into a quote :-D you addressed that anyway